cpp/CMakeLists.txt

```diff
@@ -153,16 +153,14 @@ rapids_cpm_init()
 # lags behind.
 ###
 
-# Need to make sure rmm is found before cuco so that rmm patches the libcudacxx
-# directory to be found by cuco.
+# Need CCCL, then rmm, then cuCollections, then RAFT.
+# This ensures that libraries can be overridden for testing.
```

Comment:
If this is the reason why this ordering exists in raft/cugraph*/cuml*, we probably want to document and educate rather than actually use this order.

Reply:
We decided offline that the testing ability was important and we're OK with double-bundling until rapids-core-dependencies exists.